next_generation/pint_server/tests/loadtest/loadtest.py

```diff
@@ -0,0 +1,62 @@
+from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime, timedelta
+import getopt
+import random
+import requests
+import sys
+import threading
+import time
+
+'''
+To run this program, example: python loadtest.py --num_requests 100 --duration 60
+'''
+base_url = "http://localhost:5000"
```

Comment:
I agree the base_url should be configurable.

Reply:
sure will update it